### DISCUSSION TITLE
Return end-point https://api.github.com/users/USERNAME/events/orgs/ORG

### DISCUSSION THREAD

**Discussion by fernandobortolo** [community/community#32981] [Apps, API and Webhooks]
The endpoint is returning less than specified value on per_page query param, even tweaking page and per_page params it seems to not work properly. Ex: per_page=100 and page=1 returns only 44 items,. There is something that can be done? I am sure that this org has more than 100 events in the last 90 days

**Comment by byrneh** [comment]
@fernandobortolo  how sure are you that your user has more than 44 events in this org in the last 90 days? What is the oldest event? if you create a new event  do you now see 45 items?

**Reply by fernandobortolo** [reply]
The oldest event is 2019, no, if i create a new event i still see 44 events.
